guides/common/modules/proc_configuring-logging-to-journal-or-file-based-logging.adoc

```diff
@@ -0,0 +1,42 @@
+[id="Configuring_Logging_to_Journal_or_File-based_Logging_{context}"]
+= Configuring Logging to Journal or File-based Logging
+
+{Project} uses file-based logging by default.
+You can use the `{foreman-installer}` command to reconfigure logging.
+
+.Procedure for configuring logging with Journal
+* Enter the following `{foreman-installer}` command to configure logging to the `journald` service:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} \
+--foreman-logging-layout pattern \
+--foreman-logging-level info \
+--foreman-logging-type journald \
+--foreman-proxy-log JOURNAL
+----
+
+Journal now forwards log messages to `rsyslog`, and `rsyslog` writes the log messages to `/var/log/messages`.
+After this change, the log messages do not appear in `/var/log/foreman/production.log` or `/var/log/foreman-proxy.log`.
```

Comment:
Technically this depends on how journald is configured. journald can also be configured to only keep its own entries: in `/etc/systemd/journald.conf` there's a `ForwardToSyslog` setting. We don't manage that, so at best you can say by default.

However, you can also recommend the native journald interface: `journalctl`.

Some possible commands:
* `journalctl -u foreman`
* `journalctl -u foreman-proxy`
* `journalctl REQUEST=MyRequestId`

Note that the last one doesn't filter on services, so you can actually use the correlation IDs that are sent. The idea behind that feature is that a request ID in Foreman that also causes an action in foreman-proxy has all the logs together. You can use `journalctl -u foreman REQUEST=...` to just get it for a single service.

And we don't need to document journalctl here (since you did link to the RH docs below), but it will also support `--since` and `--until` and all the regular filter options.

Reply:
I think I could implement this by adding another step to the procedure labeled "To inspect the log messages:", followed by some of the details you described.

I'll start working on this, but let me know if you think I've misinterpreted what you wrote here.

Reply:
I think a "Reading logs" procedure is useful and makes sense to include.

Reply:
I added steps with information on how to inspect the logs.